optimum/utils/preprocessing/task_processors_manager.py

```diff
@@ -35,7 +35,7 @@ class TaskProcessorsManager:
     }
 
     @classmethod
-    def get_task_processor_class_for_task(cls, task: str) -> Type:
+    def get_task_processor_class_for_task(cls, task: str) -> Type["TaskProcessor"]:
```

Comment:
Could we maintain the same style as in other funds?
```suggestion
    def get_task_processor_class_for_task(cls, task: str) ->"TaskProcessor":
```

Reply:
but one returns a constructor and the other returns an instance

Reply:
Got it, thanks for clarifying!